### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-03-07

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#141] [General]
Previous agenda: #139

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
      - [ ] https://github.com/space-ros/demos/pull/21
      - [ ] https://github.com/space-ros/docker/pull/135
- [x] Disposition4 outstanding issues from [last release](https://github.com/orgs/space-ros/projects/3/views/6)
- [x] Review status of [next release](https://github.com/orgs/space-ros/projects/3/views/8)
- [x] Open next meeting discussion

**Comment by xfiderek** [comment]
I'd like to demo my solution for https://github.com/space-ros/space-ros/issues/57, something that I said I will focus on in this release :). It should take 5-10 minutes. I have a working solution and I want to get your feedback. Btw, should we reopen that issue?

**Reply by Bckempa** [reply]
Demo looks great and issue #57 was reopened to track this work

**Comment by Bckempa** [comment]
The spellcheck issue (https://github.com/space-ros/space-ros/issues/80) received a [comment](https://github.com/space-ros/space-ros/issues/80#issuecomment-1936218616) about using (pre-commit)[https://pre-commit.com] to manage lints before things hit CI which spawned a discussion (https://github.com/space-ros/space-ros/discussions/142) about how this would work in practice.

**Reply by Bckempa** [reply]
I'm added my initial reply to the discussion. @EzraBrooks (and anyone else who cares to) please jump in on that.

**Comment by Bckempa** [comment]
Opened next meeting agenda at https://github.com/space-ros/space-ros/discussions/144
